src/linagora.esn.contact/app/app.routes.js

```diff
@@ -82,15 +82,23 @@ require('./services/contact-configuration.service.js');
         })
         .state('contact.addressbooks.show', {
           url: '^/contact/show/:bookId/:bookName/:cardId',
-          views: {
-            'main@contact': {
-              template: require('./contact/show/contact-show.pug'),
-              controller: 'ContactShowController',
-              resolve: {
-                domain: routeResolver.session('domain'),
-                user: routeResolver.session('user')
-              }
+          params: {
+            previousState: 'contact.addressbooks'
+          },
+          resolve: {
+            modalInstance: function($modal) {
+              return $modal({
+                template: '<contact-show />',
+                keyboard: false,
```

Comment:
Btw what does this option do?

Reply:
prevent closing the modal using the keyboard

Reply:
That's not nice for users without a mouse :jack_o_lantern:.